core/src/main/java/com/testaem/aem/core/utills/path/PathConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.testaem.aem.core.utills.path;
+
+public enum PathConstants {
```

Comment:
Quite original and cunning!) But... makes a little sense. 
Just
```
public class Constants {
      private Constans() {
         // To prevent instance creation
      }

     public static final CONST_1 = "...";
     public static final CONST_2 = "...";

}
```
... would do

Reply:
okok, corrected